<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mailboxClass</class>
 <widget class="QMainWindow" name="mailboxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mailbox</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="new_message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouveau message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mailbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Boite de reception</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_mailbox" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>531</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>boite de reception</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>EN DEV</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_send_message" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>441</width>
      <height>353</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_from">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FROM :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_from">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_to">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_to">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>TO :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_subject">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_subject">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_send">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="content_message">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>311</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
